frontend/app/work_packages/directives/inplace_editor/inplace-editor-edit-pane-directive.js

```diff
@@ -40,8 +40,27 @@ module.exports = function(
     templateUrl: '/templates/work_packages/inplace_editor/edit_pane.html',
     controllerAs: 'editPaneController',
     controller: function($scope, WorkPackageService) {
+      var vm = this;
+      var acknowledgedValidationErrors = ['required', 'number'];
+
       this.submit = function(notify) {
         var fieldController = $scope.fieldController;
+
+        if (vm.editForm.$invalid) {
+          var detectedViolations = [];
+          acknowledgedValidationErrors.forEach(function(error) {
+            if (vm.editForm.$error[error]) {
+              detectedViolations.push(I18n.t('js.inplace.errors.' + error, {
+                field: fieldController.getLabel()
+              }));
+            }
+          });
+          if (detectedViolations.length) {
```

Comment:
Urgh... do they still do it like C in JavaScript?

I would have written `> 0`, but I guess this is optional. Do we have a styleguide for such things?